kodaksmarthome-hk/_version_.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
Do not create duplication of ``kodaksmarthome``.
Please do the changes on``kodaksmarthome`` instead creating ``kodaksmarthome-hk``

Reply:
Got it. New to GitHub and was trying something.